pages/badges/badge-single.jsx

```diff
@@ -234,16 +234,26 @@ var BadgePage = React.createClass({
     );
   },
 
+  getShareCodes: function() {
+    var url = encodeURIComponent(window.location.toString());
+    var msg = encodeURIComponent(`I earned the Mozilla ${this.state.badge.title} badge`);
+
+    return (
+      <div className="social-share">
+        <a href={`https://twitter.com/home?status=${msg + encodeURIComponent(': ') + url}`} target={'_blank'}>facebook</a>
+        <a href={`http://www.facebook.com/sharer.php?u=${url}&t=${msg}`} target={'_blank'}>twitter</a>
```

Comment:
lol, nice.

oh yeah I'm pretty sure that was supposed to be https and I forgot to add the `s` there.